CRM/Upgrade/Incremental/php/FiveFourteen.php

```diff
@@ -103,4 +105,71 @@ public static function rebuildCaseActivityView($ctx) {
     return TRUE;
   }
 
+  /**
+   * This task adds the category column to case types. The category is an option
+   * value that can either be Workflow or Vacancy.
+   *
+   * @param CRM_Queue_TaskContext $ctx
+   * @return bool
+   */
+  public static function createCaseTypeCategoryField($ctx) {
+    self::addCategoryColumnToCaseType();
+    self::createCaseTypeCategories();
+    self::setDefaultCategoriesForExistingCaseTypes();
+  }
+
+  /**
+   * Adds a new category column to the case type entity. This column references
+   * option values.
+   */
+  private static function addCategoryColumnToCaseType() {
+    CRM_Core_DAO::executeQuery('ALTER TABLE civicrm_case_type
```

Comment:
use the add field function rather than this

Reply:
@eileenmcnaughton can you please point me to the specific "add field" method you are referring too ? Tried to find an example in the upgraders and couldn't find any.

Reply:
@reneolivo I think she means addColumn in Upgrade_Incremental_Base

Reply:
thanks a lot @demeritcowboy, found it!